arrow/locales.py

```diff
@@ -1717,7 +1799,7 @@ class MacedonianLocale(SlavicBaseLocale):
     ]
 
 
-class GermanBaseLocale(Locale):
+class GermanBaseLocale(Locale, metaclass=ABCMeta):
```

Comment:
Added `ABCMeta`

Reply:
Mind explaining the purpose of a metaclass?

Reply:
Can't specific reason... 😅
I'll remove it and discuss in another PR